pkg/ui/chooser/embed/ns/index.html

```diff
@@ -0,0 +1,32 @@
+<html>
+<head>
+    <title>Namespaces</title>
+    <link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.css" />
+    <script src="https://code.jquery.com/jquery-3.7.1.min.js"   integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo="   crossorigin="anonymous"></script>
+    <script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.js"></script>
```

Comment:
I'm adding these JS and CSS into the repo, since G-R internal users probably cannot fetch from public cdn sites.

Reply:
okay I'll fix it